composer.json

```diff
@@ -59,14 +60,14 @@
             "Composer\\Config::disableProcessTimeout",
             "@cs",
             "@deduplicate",
-            "@analyze",
             "@inspect",
+            "@analyze",
```

Comment:
Why this change?

Reply:
The same as devkit:
https://github.com/codeigniter4/devkit/blob/d4ac84b92ede7a7eb5dc21b6aa0af7b4001efd5d/composer.json#L59-L66